packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -133,6 +133,17 @@ watch(
   },
   { deep: true },
 );
+
+watch(
+  [props.errorMessage, inputElement],
+  () => {
+    if (!inputElement.value) return;
+    // by using setCustomValidity, the ValidityState will turn invalid
+    // as long as it's not an empty string
+    inputElement.value.setCustomValidity(props.errorMessage || "");
```

Comment:
```suggestion
    if (!inputElement.value || !props.errorMessage) return;
    // by using setCustomValidity, the ValidityState will turn invalid
    // as long as it's not an empty string
    inputElement.value.setCustomValidity(props.errorMessage);
```

The custom error should only be set if a message exists, right?

Reply:
it's important that it is reset again when props.errorMessage is removed / reset to empty string 

that change would be a problem because
![Uploading image.png…]()

Reply:
Ah okay so calling `setCustomValidity` with an empty string will reset the error and make the form valid again?

Reply:
yep :) as long as no other error is still active

Reply:
Nice 👍